### DISCUSSION TITLE
Conan FastDDS 3.x.x

### DISCUSSION THREAD

**Discussion by ebassi00** [eProsima/Fast-DDS#5368] [Support]
Hi everyone,
as the title suggests, I'm here to ask you if and when you were planning to publish version 3.x.x of FastDDS on Conan. I work for a company that uses FastDDS 2.x.x and we would need version 3.x.x to check incoming/outgoing (complex) messages between participants on different topics via FastDDSSpy / FastDDSMonitor. I specify that we use FastDDS via the Conan version so we have no possibility of compiling the 3.x.x by hand.

**Comment by MiguelCompany** [comment]
This is not something we manage at eProsima.
All contributions for the recipes [here](https://github.com/conan-io/conan-center-index/tree/master/recipes/fast-dds) have been from external contributors.

**Reply by ebassi00** [reply]
Thank you for your reply.
Do you have any contacts with someone who previously pushed on ConanCenter?

**Reply by rsanchez15** [reply]
As @MiguelCompany said, we have never pushed Fast DDS releases to ConanCenter, but please email me at raul@eprosima.com to continue the conversation and to let us know your specific needs.
